index.html

```diff
@@ -1,44 +1,226 @@
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="1">See Project</button>
+                </div>
+            </div>
+            <div class="works-2">
+                <div class="project-image"><img class="image" src="images/header/second.png" alt="my first project"></div>
+                <div class="projects">
+                    <h2 class="project-tittle">Multi-Post Stories</h2>
+                    <ul class="project-info">
+                        <li class="client">FACEBOOK</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>Full Stack Dev</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>2015</li>
                     </ul>
-                </section>
+                    <p class="primary-text">Experimental content creation feature that allows users to add to an
+                        existing
+                        story over the course of a day without spamming their friends.</p>
+                    <ul class="categories">
+                        <li>html</li>
+                        <li class="ruby">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="2">See Project</button>
+                </div>
+            </div>
+            <div class="works">
+                <div class="project-image"><img class="image" src="images/header/third.png" alt="my first project"></div>
+                <div class="projects">
+                    <h2 class="project-tittle">Facebook 360</h2>
+                    <ul class="project-info">
+                        <li class="client">FACEBOOK</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>Full Stack Dev</li>
+                        <li><img class="imgPoint" src="images/header/Counter.png" alt="point"></li>
+                        <li>2015</li>
+                    </ul>
+                    <p class="primary-text">Exploring the future of media in Facebook's first Virtual Reality app; a
+                        place
+                        to discover and enjoy 360 photos and videos on Gear VR.</p>
+                    <ul class="categories">
+                        <li>html</li>
+                        <li class="ruby">Ruby on rails</li>
+                        <li>css</li>
+                        <li>javaScript</li>
+                    </ul><button class="button" id="3">See Project</button>
+                </div>
+            </div>
+            <div class="works-2">
+                <div class="project-image"><img class="image" src="images/header/desktop/d.png" alt="my first project"></div>
```

Comment:
- [REQUIRED] As the[ last code reviewer suggested](https://github.com/Praveen1995-10-14/myportfolio/pull/10#discussion_r990640251) kindly fix the src property of this image tag because is not rendering the image :pray: :pray:  You could try this. src="./images/header/desktop/d.png" Please pay attention on the `./` before the path. In a local environment, the path that you put could be working but if you deploy the project you are going to have the error that the last code reviewer mentioned.

![image](https://user-images.githubusercontent.com/98363075/194714968-d37477c9-8963-44f6-b2a4-b103690ffc42.png)


- [OPTIONAL] Kindly make this change in all your tag images and background images because are not rendering. :+1: :smiley: :smiley: